cmake/modules/gluecodium/gluecodium/runGenerate.cmake

```diff
@@ -72,15 +72,17 @@ endfunction()
 function(_collect_all_files_in_single_compilation_units)
     _include_all(cpp "cpp/*.cpp")
 
-    if(APIGEN_GENERATOR STREQUAL android)
+    if(APIGEN_GENERATOR MATCHES android)
```

Comment:
Probably `android IN_LIST APIGEN_GENERATOR` is clearer

Reply:
But it's not a CMake list, it's comma-separated, not semicolon-separated.